Engine/DataFeeds/ApiDataProvider.cs

```diff
@@ -88,19 +88,19 @@ public static bool IsOutOfDate(Resolution resolution, string filepath)
         /// <summary>
         /// Attempt to download data using the Api for and return a FileStream of that data.
         /// </summary>
-        /// <param name="key"></param>
-        /// <param name="symbol"></param>
-        /// <param name="date"></param>
-        /// <param name="resolution"></param>
+        /// <param name="filePath">The path to store the file</param>
+        /// <param name="symbol">The symbol we are downloading</param>
+        /// <param name="date">The date of the data</param>
+        /// <param name="resolution">The resolution of the data</param>
         /// <returns>A FileStream of the data</returns>
-        private FileStream DownloadData(string filepath, Symbol symbol, DateTime date, Resolution resolution)
+        private FileStream DownloadData(string filePath, Symbol symbol, DateTime date, Resolution resolution)
```

Comment:
We don't need the arguments `Symbol symbol, DateTime date, Resolution resolution` anymore really, think we should remove them if possible